NOTES-NONSTOP.md

```diff
@@ -57,7 +57,12 @@ relating to `atexit()` processing when a shared library is unloaded and when
 the program terminates. This limitation applies to all OpenSSL shared library
 components.
 
-A resolution to this situation is under investigation.
+A control has been added as of 3.3.x to disable, by default, calls to `atexit()`
+within the `libcrypto` builds (specifically in `crypto/init.c`). This variable,
+`OPENSSL_NO_DLL_ATEXIT_HANDLERS=1` is set by default for NonStop builds. If you
```

Comment:
I would drop the DLL part of this name. This is something that could be used on many platforms, and DLL is very platform specific.

Reply:
> I would drop the DLL part of this name. This is something that could be used on many platforms, and DLL is very platform specific.

True, but this only happens when atexit() code is called from a DLL, so is DLL-specific.

Reply:
I agree with @mattcaswell here.  I understand it only happens on nonstop in a DLL, but the option to disable atexit registration at build time might be generally useful outside of just this platform, and so should probably be named accordingly to prevent confusion.

Reply:
I'm going down this path now, but need a bit of help tweaking Configure. Please see the next conversation.